version/version.go

```diff
@@ -17,7 +17,7 @@ var (
 	//
 	// Version must conform to the format expected by
 	// github.com/hashicorp/go-version for tests to work.
-	Version = "1.4.3"
```

Comment:
I believe this got missed last release due to the frenzy of trying to revert a breaking change and release again ASAP